### PR TITLE
Feature to create ILB sets

### PR DESCRIPTION
**Pull request by paxperscientiam**
This PR adjusts event listeners and targeting such that an Image Lightbox set is created based on the value of the data-imagelightbox attribute.

You still do as usual to call...
````js
$("a[data-imagelightbox]").imageLightbox({...});
````

The difference in usage (more like a different way of thinking about ILB):
````HTML
<a href="/img1.png" data-imagelightbox="a"></a>
<a href="/img2.png" data-imagelightbox="a"></a>

<a href="/img3.png" data-imagelightbox="b"></a>
````
When the user clicks link to image "img1.png", a set is created for each data-imagelightbox equal to, here, "a"; it will include "img1.png" and "img2.png".

Clicking the link to "img3.png" will create a separate ILB set.

In summary, the user can now have multiple galleries with their own ILB instance all on one webpage.

EDIT: grammar

### CODE REVIEW

**Comment by rejas** [comment]
Same here: I like the idea but reviewing the PR is hard when other changes are in there :-(

**Comment by paxperscientiam** [comment]
Fixing now!

**Comment by paxperscientiam** [comment]
OK bud, how we looking?

**Comment by rejas** [comment]
Still too much noise in there, you can see it for yourself when clicking on "Files changed" on top of this page....
